### PR TITLE
Fix a bug when promise chain created cycle dependency

### PR DESCRIPTION
**Pull request by andriichumak**
When used with Promise API instead of async/await, websocket-based commands would create cycle dependency and would never resolve. Now it's fixed.

Also, switched SuitestLauncher commands to unchained versions in scope of this ticket, because it was hard to debug without this changes.

### CODE REVIEW

File: lib/testLauncher/SuitestLauncher.js [hunk 1]
```diff
@@ -26,64 +28,71 @@ class SuitestLauncher {
 		try {
```
Comment (JeKachan) [inline lib/testLauncher/SuitestLauncher.js]: do we really need this try-catch? Almost all covered code will be executed inside child processes.

Comment (andriichumak) [reply lib/testLauncher/SuitestLauncher.js]: There are also calls to server and we have to show to user a meaningful message in that case.
Second try/catch is only for "finally" part - to make sure session is closed on server properly if something goes wrong.

Comment (JeKachan) [reply lib/testLauncher/SuitestLauncher.js]: what if closeSessionUnchained on 42 line will failed? We will get finally block and there closeSessionUnchained will be called twice.

Comment (JeKachan) [reply lib/testLauncher/SuitestLauncher.js]: Sorry, I was a little mistaken, I talked about try catch on 37 line

Comment (andriichumak) [reply lib/testLauncher/SuitestLauncher.js]: You're right, should have put second try somewhere after line 45. Will update.

**Pull request merged**